src/structure/structure.ts

```diff
@@ -1,16 +1,16 @@
 import { apiClient } from '../erudio';
 
 export interface Options {
-  filter: string;
-  page: number;
-  limit: number;
-  sort: string;
+  filter?: string;
+  page?: number;
+  limit?: number;
+  sort?: string;
 }
 
 export class Structure {
   private base_url = 'http://edtech-structure-service.dev.svc.cluster.local';
```

Comment:
@JERILJOSE96 what about prod env? How we're going to switch between them? I think that's an important feature to have.